combat-trainer.lic

```diff
@@ -3401,6 +3404,16 @@ class GameState
 
     @whirlwind_trainables = settings.whirlwind_trainables
     echo("  @whirlwind_trainables: #{@whirlwind_trainables}") if $debug_mode_ct
+    
+    @flame_inner_fire_threshold = settings.barb_combat_inner_fire_threshold
+    echo("  @flame_inner_fire_threshold: #{@flame_inner_fire_threshold}") if $debug_mode_ct
+
+    if @whirlwind_trainables && @charged_maneuvers
+      @flame_for_expertise = false
+    else
+      @flame_for_expertise = true
+    end
```

Comment:
Thoughts on condensing this to one line? I don't have a strong opinion either way.
```suggestion
    @flame_for_expertise = !(@whirlwind_trainables && @charged_maneuvers)
```